sql/npc_list.sql

```diff
@@ -12714,9 +12714,9 @@ INSERT INTO `npc_list` VALUES (17212100,'Elrica','Elrica',0,0.000,0.000,0.000,0,
 INSERT INTO `npc_list` VALUES (17212101,'Terounalivet','Terounalivet',0,0.000,0.000,0.000,0,40,40,0,0,0,6,2075,0x0100060353105320533053405350006000700000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17212102,'qm1','???',0,309.600,2.600,324.000,32774,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
 INSERT INTO `npc_list` VALUES (17212103,'Cavernous_Maw','Cavernous Maw',0,462.628,0.029,479.280,32769,40,40,0,0,96,0,3,0x0000140900000000000000000000000000000000,2,'WOTG',1);
-INSERT INTO `npc_list` VALUES (17212104,'apocalypse','     ',0,-217.283,97.267,461.827,32774,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,NULL,1);
```

Comment:
need the correct retail ID for the apoc NPC before I can merge this, because it would break relic quest otherwise.

possibilities include: 
- it being the blank right between sparkles and the lyco (unlikely)
- it having been correct but our sparkles are off by one (the others all seem to have a 1 npc gap between lyco and sparkles, so I dunno)
- actually some other NPC slot in the zone

Reply:
Value of NPC is a duplicate of 17212087 which is the actual one being used by relic.lua for that zone's relic quest

```INSERT INTO `npc_list` VALUES (17212087,'relic','     ',146,-217.279,97.281,461.636,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,NULL,1);```

Correct information for 17212104 was capture by @UynGH  (See https://github.com/KnowOne134/NPC-MOB-Logger/blob/master/Tables/North%20Gustaberg.lua) which was the one used to update sql line on this PR